server/memphis_handlers_schemas.go

```diff
@@ -41,6 +49,37 @@ func validateProtobufContent(schemaContent string) error {
 	return nil
 }
 
+func generateProtobufDescriptor(schemaName string, schemaVersionNum int, schemaContent string) ([]byte, error) {
+	filename := fmt.Sprintf("tmp_proto_%s_%s", schemaName, schemaVersionNum)
```

Comment:
add .proto this is important for what happens in the SDK side

Reply:
How? these files never get to any SDK, they are also deleted few lines after.

Reply:
I know but when you compiling the file the name is encoded into the descriptor and the SDK count on it